src/App.vue

```diff
@@ -1,12 +1,8 @@
 <template>
   <Theme :lang="lang" :theme="theme">
-    <WidgetMainSmall
-      title="Up To 60 months interest-free."
-      subtitle="Indicative payments. Ts&Cs apply."
-      button-primary-label="Explore"
-      button-close-label="Close"
+    <component
+      :is="currentWidget"
```

Comment:
I wonder if we could change this so that we have two App components. `App-au` and `App-nz`, then each of these won't have to bloat its bundle size with Widgets that aren't applicable to each language. 
Do you think that's doable and feasible?